docs/protocol/ISM/multisig-ISM.mdx

```diff
@@ -1,3 +1,40 @@
 # Multisig ISM
 
 The `MultisigISM` is one of the most commonly used ISM types. These ISMs verify that `m` of `n` [Validators](../agents/validators.mdx) have attested to the validity of a particular interchain message.
+
+## Multisig Module Types
+
+Based on the following module types, the relayer will encode the appropriately formatted metadata:
+
+```solidity
+enum Types {
+    ...
+    MERKLE_ROOT_MULTISIG,
+    MESSAGE_ID_MULTISIG,
+    ...
+}
+```
+
+For the aforementioned module Types, the following metadata formats are satisfied by the relayer for use by the ISMs:
+
+### MessageIdMultisigIsmMetadata
+
+```solidity
+Format of metadata:
+[   0:  32] Origin merkle tree address
+[  32:  64] Signed checkpoint root
+[  64:  68] Signed checkpoint index
+[  68:????] Validator signatures (length := threshold * 65)
```

Comment:
I know this is just taken from the existing source doc but imo this isn't the most clear
can we maybe add some stuff about byte indexed

Reply:
yep forsure– was hasty